### PR TITLE
Add info for ace editor

### PR DESCRIPTION
**Pull request by hristo-vrigazov**
Added information for Ace editor, which is the editor Cloud9 runs on. Could be helpful since it is open source.

### CODE REVIEW

**Comment by styfle** [comment]
Hi Hristo, thanks for the PR!

Can you remove the period so it more closely matches the line below (Eclipse Che) and also link to the GitHub and Docker repos?

**Comment by hristo-vrigazov** [comment]
I have not included a Docker repo, since it does not have a standalone official repo, but added Github and changed the phrasing.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A curated list of awesome online implementations of integrated development envir
...
+* [AWS Cloud9](https://aws.amazon.com/cloud9) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/c9/core) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/r/cloud9/workspace/) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run any language on a full VM complete with a terminal with on [Ace Editor](https://ace.c9.io/)  [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/ajaxorg/ace)
```
Comment (styfle) [inline README.md]: Change "with on" to "built on"

**Comment by styfle** [comment]
Perfect! 💯

**Pull request merged**